pkg/pillar/cmd/msrv/handlers.go

```diff
@@ -787,3 +805,73 @@ func (msrv *Msrv) handleActivateCredentialPost() func(http.ResponseWriter, *http
 		w.Write(out)
 	}
 }
+
+func (msrv *Msrv) handleNetworkStatusMetrics() func(http.ResponseWriter, *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		msrv.Log.Tracef("networkStatusMetricsHandler.ServeHTTP")
+
+		// Fetch network status data
+		devNetStatusObj, err := msrv.subDeviceNetworkStatus.Get("global")
+		if err != nil {
+			http.Error(w, http.StatusText(http.StatusNoContent), http.StatusNoContent)
+			return
+		}
+		devNetStatus := devNetStatusObj.(types.DeviceNetworkStatus)
+
+		// Fetch network metrics data
+		metricsObj, err := msrv.subNetworkMetrics.Get("global")
+		if err != nil {
+			http.Error(w, http.StatusText(http.StatusNoContent), http.StatusNoContent)
+			return
+		}
+		metrics := metricsObj.(types.NetworkMetrics)
+
+		// List only physical interfaces
+		devList := make([]networkStatusMetrics, 0)
+		for _, p := range devNetStatus.Ports {
+			var metric *types.NetworkMetric
+			if !p.IsL3Port || p.IfName == "" {
```

Comment:
I think you can remove `!p.IsL3Port` to be in line with changes submitted yesterday: https://github.com/lf-edge/eve/pull/4694